### ISSUE TITLE
tmap_leaflet() not working in v4

### ISSUE THREAD

**Issue by Robinlovelace** [r-tmap/tmap#748]
As outlined in what should be a more-or-less reproducible example here: https://github.com/ITSLeeds/netvis/issues/7

**Comment by mtennekes** [comment]
You can check if it exists, and if not add @export
As workaround you should be able to do `x = print(tm)` in view mode.

**Comment by Robinlovelace** [comment]
Will check, thanks Martijn! Good one for the hackathon?

**Comment by Robinlovelace** [comment]
Fixed. More docs here: https://github.com/r-tmap/tmap/pull/750

**Comment by Robinlovelace** [comment]
Actually no, that only worked after `devtools::load_all()`, re-opening. If anyone can reproduce let us know.

**Comment by Robinlovelace** [comment]
There's no tmap_leaflet in here it seems despite the `@export` tag being there, right?

https://github.com/r-tmap/tmap/tree/v4/man

**Comment by Robinlovelace** [comment]
https://github.com/r-lib/roxygen2/issues/852 related?

**Comment by Nowosad** [comment]
Have you run `devtools::document()` first?

**Comment by Robinlovelace** [comment]
> Have you run `devtools::document()` first?

Yep, several times...

**Comment by Nowosad** [comment]
@Robinlovelace Close?

**Comment by Robinlovelace** [comment]
Fixed in #751

**Comment by spedunkler** [comment]
I'm not seeing `tmap_leaflet()` working when an html document is knitted from an .rmd, which is when I've used it in the past.  For instance, the sample code below produces no map in the html view, though it does work when running the code chunk in the RStudio IDE.
```
library(tmap)
tmap_mode("view")
map = tm_shape(World) + tm_polygons()
tmap_leaflet(map, show = TRUE)
```
